mir_connector/inorbit_mir_connector/src/mir_api/mir_api_v2.py

```diff
@@ -59,7 +59,7 @@ def __init__(
             base_url=self.mir_api_base_url,
             auth=self._auth,
             default_headers={"Accept-Language": "en_US"},
-            timeout=10,
+            timeout=30,  # Increased timeout to handle slower responses
```

Comment:
```suggestion
            timeout=30,
```
Is this comment referring to the previous value of this parameter? AI often does that. if so I'd remove it.